site/content/docs/5.3/examples/e-shop/index.html

```diff
@@ -0,0 +1,1568 @@
+
+                    <input type="radio" id="bad1" name="rating1" value="2" class="visually-hidden">
+                    <label for="bad1" title="Bad"></label>
+
+                    <input type="radio" id="mixed1" name="rating1" value="3" class="visually-hidden" checked>
+                    <label for="mixed1" title="Mixed"></label>
+
+                    <input type="radio" id="good1" name="rating1" value="4" class="visually-hidden">
+                    <label for="good1" title="Good"></label>
+
+                    <input type="radio" id="excellent1" name="rating1" value="5" class="visually-hidden">
+                    <label for="excellent1" title="Excellent"></label>
+                  </fieldset>
+                  <span class="visually-hidden">rated 3 stars out of 5</span>
+                  <p class="mb-3 fw-bold d-inline-flex align-middle">
+                    <svg class="border border-1 rounded-circle me-2 bg-dark" width="1.25rem" height="1.25rem" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false">
+                      <defs>
+                        <linearGradient id="Gradient1">
+                          <stop offset="0%" stop-color="#0400ff"/>
+                          <stop offset="100%" stop-color="#fff800"/>
+                        </linearGradient>
+                      </defs>
+                      <circle cx="10" cy="10" r="10" fill="url('#Gradient1')"/>
+                    </svg>
+                    5 colors
+                  </p>
+                  <p class="mb-0 pt-md-1">
+                    <span class="d-block mb-1 fw-bold">From</span>
+                    <span class="fs-4 fw-bold text-primary">2 500 350 FCFA</span><br>
+                    <span class="visually-hidden">instead of </span>
+                    <span class="text-decoration-line-through">3 000 000 FCFA</span>
+                  </p>
+                </div>
+              </div>
+              <div class="row">
+                <div class="offset-5 col-7">
+                  <form class="mb-3">
+                    <div class="form-check">
+                      <input type="checkbox" class="form-check-input" id="phoneCompare1">
+                      <label for="phoneCompare1" class="form-check-label">Compare <span class="visually-hidden">xxxxx xxxxxx xx xxx xxx to other selected smartphones</span></label>
+                    </div>
+                  </form>
+                  <a href="#" class="btn btn-info">More details <span class="visually-hidden">about xxxxx xxxxxx xx xxx xxx</span></a>
+                </div>
+              </div>
+            </div>
+          </li>
+
+          <li class="card border-0 pt-4 pt-md-4">
+            <h3 class="visually-hidden">xxxx xxx</h3>
+            <div class="d-flex mt-md-3 mb-2 align-items-end justify-content-between border-bottom border-danger fw-bold">
+              <div class="d-flex mt-md-3">
```

Comment:
Not a big fan of those `.mt-md-3` and `mt-xl-3` since they are quite too manual imo. At least we get the same rendering as the design.